tools/project_manager/bad_bytes_explainer.php

```diff
@@ -153,6 +153,7 @@ function show_a_table_of_bads($desired_badness)
         <table class='basic striped'>
         <tr>
             <th>" . _("Raw") . "</th>
+            <th>" . _("Codepoints") . "</th>
             <th>" . _("Bytes") . "</th>
```

Comment:
I would prefer we put the Bytes column ahead of the Bytes column. I think we should always show both columns instead of jumping through the hoops with the colspans (for HTML entity fields the Codepoints field would be empty). We should retain the 'U+' syntax for the codepoint column so it's clearer these are Unicode character values.

Reply:
Done this.